cypress/e2e/immutableDatabaseTests/linkCheck.ts

```diff
@@ -0,0 +1,104 @@
+import { setTokenUserViewPort } from '../../support/commands';
+
+function formatPaths(paths: string[]) {
+  let formattedPaths = '';
+  paths.forEach((path) => {
+    formattedPaths += '\n- ' + path;
+  });
+  return formattedPaths;
+}
+
+describe('Find broken anchor links', () => {
+  setTokenUserViewPort();
+
+  function checkUrls(path: string, selector: string) {
+    let brokenUrls = [];
+    cy.visit(path);
+    cy.get(selector)
+      .find('a')
+      .each((url) => {
```

Comment:
Suggest a different variable name than `url`.   Here, we're iterating over the elements that were found, the URL is extracted from each further down.